dipy/reconst/tests/test_dsi.py

```diff
@@ -78,13 +80,18 @@ def test_multib0_dsi():
     new_bvecs = np.concatenate([gtab.bvecs, np.zeros((1, 3))])
     new_bvals = np.concatenate([gtab.bvals, [0]])
     new_gtab = gradient_table(new_bvals, new_bvecs)
-    DS = DiffusionSpectrumModel(new_gtab)
-    sphere = get_sphere('symmetric724')
-    DSfit = DS.fit(new_data)
-    PDF = DSfit.pdf()
-    assert_equal(new_data.shape[:-1] + (17, 17, 17), PDF.shape)
-    assert_equal(np.alltrue(np.isreal(PDF)), True)
+    ds = DiffusionSpectrumModel(new_gtab)
+    sphere = get_sphere('repulsion724')
+    dsfit = ds.fit(new_data)
+    pdf = dsfit.pdf()
+    odf = dsfit.odf(sphere)
+    assert_equal(new_data.shape[:-1] + (17, 17, 17), pdf.shape)
+    assert_equal(np.alltrue(np.isreal(pdf)), True)
+    from dipy.viz import fvtk
```

Comment:
Sorry: I commented on the commit, instead on the diff. 

We should probably remove these viz calls before merging, correct?